components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -31,12 +31,29 @@ type GardenerConfig {
     targetProvider: String
     targetSecret: String
     diskType: String
-    zone: String
-    cidr: String
+    workercidr: String
```

Comment:
Let's keep it consistent: `workerCidr` should be here (since `vnetCidr` is already present in Azure - specific config)